nixos/tests/nextcloud/basic.nix

```diff
@@ -1,27 +1,27 @@
-args@{ pkgs, nextcloudVersion ? 22, ... }:
+{ name, pkgs, testBase, system,... }:
 
-(import ../make-test-python.nix ({ pkgs, ...}: let
-  adminpass = "notproduction";
-  adminuser = "root";
-in {
-  name = "nextcloud-basic";
+with import ../../lib/testing-python.nix { inherit system pkgs; };
+runTest ({ config, ... }: {
+  inherit name;
   meta = with pkgs.lib.maintainers; {
-    maintainers = [ globin eqyiel ];
+    maintainers = [ globin eqyiel ma27 ];
```

Comment:
We should probably drop the inactive maintainers

Reply:
I don't really want to offboard people on my own, even if it's just for a single package. That includes inhibiting PRs to await feedback on this (with a reasonable timeout). While (temporarily) revoking commit access is fine, removing people from all packages seems a little more painful (if I'm gone for a year, I don't want to re-add myself everywhere!).

In other words, I don't think ad-hoc actions help much here, we'd need a defined process for that.

For now, a maintainer doesn't have too much power & responsibilities, though it may be desirable, so I don't see the benefit of this as of now.

Reply:
I am still watching, just don't have enough free time to actively work on larger issues and generally don't want to interfere with low quality feedback. If there were urgent matters to be worked on, I'd still do so.